fix: update module name to use github url (#280)
elastic/apm-aws-lambda: TESTING.md

# APM AWS Lambda extension Testing

## Testing unreleased versions

> If you need to test an unreleased version you can compile the package and it's dependencies and make them available as AWS Lambda Layer.

Reviewer: ```suggestion
If you need to test an unreleased version you can compile the package and its dependencies and make them available as AWS Lambda Layer.
```

### Compile package and dependencies

To run an unreleased version of this extension, you will need to ensure that your build architecture matches that of the Lambda execution environment by compiling with `GOOS=linux` and `GOARCH=amd64` if you are not running in a Linux environment.

> To build the extension into the `bin/extensions` folder, run the following commands.

Reviewer: ```suggestion
To build the extension in the `bin/extensions` folder, run the following commands.
```

 --zip-file  "fileb://extension.zip"
```

> The out from the above command will include a `LayverVersionArn` field, which contains the unique string identifier for your layer.  The will look something like the following.

Reviewer: ```suggestion
The output from the above command will include a `LayerVersionArn` field, which contains the unique string identifier for your layer.  The will look something like the following.
```
